src/engine/compiler/SinglePassCompiler.v3

```diff
@@ -1457,6 +1536,9 @@ class SinglePassCompiler(xenv: SpcExecEnv, masm: MacroAssembler, regAlloc: RegAl
 		var pc = it.pc - start_pos;
 		instrTracer.instr_width = Opcodes.longestName + 1;
 		instrTracer.putPcAndInstr(OUT, module, func, pc, orig);
+		OUT.puts("JIT code: ");
+		masm.printCodeBytes(OUT);
+		OUT.ln();
```

Comment:
I think this makes debugging easier, since we can see the exact assembly generated for each opcode